docs/source/user_manual/matplotlib_to_chaco.rst

```diff
@@ -0,0 +1,55 @@
+.. _matplotlib2chaco:
+
+##############################
+Matplotlib To Chaco Cheatsheet
+##############################
+As mentioned in the :ref:`Chaco Tutorial <tutorial>`,
+`matplotlib <https://matplotlib.org/>`_ and Chaco have different
+approaches to plotting (script-oriented vs. application oriented). However, new
+users of chaco who have worked with matplotlib in the past may find it useful
+to think about mapping "equivalent" code between the two.
+
+The following table lists various plotting related operations one might want to
+perform along with code for doing that using either Chaco or Matplotlib. For
+the Matplotlib code ``ax`` repesents the Axes that what would result from
+running  ``fig, ax = plt.subplots()`` after importing
+``import matplotlib.pyplot as plt``. For more details see the
+`matplotlib User's Guide <https://matplotlib.org/stable/users/index.html>`_.
+For Chaco code, ``p`` is the :class:`~.Plot` instance.
+
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|          What          |                matplotlib                |                                 Chaco                                  |
++========================+==========================================+========================================================================+
+|       Plot lines       |             `ax.plot(x, y)`              |                          `p.plot(('x', 'y'))`                          |
```

Comment:
it's not straightforward for people to know what the string `'x'` and `'y'` mean. It would be useful if you expanded on the earlier paragraph by adding some more information on instantiating the `Plot` object with the traits `x` and `y`, which we are referring to here.